src/aks-preview/linter_exclusions.yml

```diff
@@ -15,11 +15,6 @@ aks create:
     enable_workload_identity:
       rule_exclusions:
         - option_length_too_long
-aks delete:
-  parameters:
-    ignore_pod_disruption_budget:
-      rule_exclusions:
-        - option_length_too_long
```

Comment:
Need to keep this to bypass the lint error for the `aks delete` command. The command is auto generated according to the SDK.

Reply:
I see.